office/zathura/Portfile

```diff
@@ -54,6 +54,29 @@ For handling additional file types, consider installing the various
 zathura-plugin-* ports (port search zathura-plugin).
 "
 
+subport ${name}-docs {
+    set python.version 39
+    set python.dot_version 3.9
```

Comment:
From the python portgroup, I believe this can be generalized as

`set python.dot_version {[string index ${python.version} 0].[string range ${python.version} 1 end]}`